src/seedu/addressbook/data/person/Person.java

```diff
@@ -14,8 +14,10 @@
     private Phone phone;
     private Email email;
     private Address address;
+    private int sequenceNo;
 
     private final UniqueTagList tags;
+    private static int nextSeqNo = 0;
```

Comment:
Try to cherry pick and submit if you have multiple activities done. Else you can do them in separate branches to avoid sending in multiple activities in a single PR.